lsif-java/src/main/scala/com/sourcegraph/lsif_java/buildtools/GradleJavaCompiler.scala

```diff
@@ -98,6 +99,43 @@ case class GradleJavaCompiler(languageVersion: String, javacPath: Path) {
       )
       .toFile
       .setExecutable(true)
+
+    // For compile{Test}Kotlin when using jvm toolchains, we need to have access
+    // to JDK internals found in <java-installation-path>/lib in JDK 9+,
+    // as well as <java-installation-path>/jre/lib in JDK <=8, else we get
+    // "no class roots are found in the JDK path" from the compile{Test}Kotlin tasks.
+    // https://docs.oracle.com/en/java/javase/12/migrate/index.html#JSMIG-GUID-A78CC891-701D-4549-AA4E-B8DD90228B4B
+
+    val copyFiles =
+      (source: Path, destination: Path) => {
+        Files
+          .walk(source)
+          .forEach(t => {
+            val destPath = destination.resolve(source.relativize(t))
+            try {
+              Files.copy(t, destPath)
+            } catch {
+              case _: NoSuchFileException =>
+                return
+            }
+          })
+      }
+
```

Comment:
```suggestion
    val copyFiles =
      (source: Path, destination: Path) => {
        Files
          .walk(source)
          .forEach(t => {
            val destPath = destination.resolve(source.relativize(t))
            try {
              Files.copy(t, destPath)
            } catch {
              case _: NoSuchFileException =>
                return
            }
          })
      }

    // For compile{Test}Kotlin when using jvm toolchains, we need to have access
    // to JDK internals found in <java-installation-path>/lib in JDK 9+,
    // as well as <java-installation-path>/jre/lib in JDK <=8, else we get
    // "no class roots are found in the JDK path" from the compile{Test}Kotlin tasks.
    // https://docs.oracle.com/en/java/javase/12/migrate/index.html#JSMIG-GUID-A78CC891-701D-4549-AA4E-B8DD90228B4B
```

Nit: Please put the comment closer the code it pertains to.